bentoml/deployment/aws_lambda/utils.py

```diff
@@ -221,15 +221,15 @@ def init_sam_project(
         bundled_files = os.listdir(
             os.path.join(function_path, 'bundled_pip_dependencies')
         )
-        has_bentoml_bundle = False
+        has_bentoml_bundle = False  # noqa # pylint: disable=unused-variable
         for index, bundled_file_name in enumerate(bundled_files):
```

Comment:
cc @yubozhao, should we just remove `has_bentoml_bundle` and related code?

Reply:
Yep.  We no longer need it

Reply:
Got it.

@kautukkundan could you help remove l224, as well as L229-232 where `has_bentoml_bundle` was set? this variable is not used anymore due to refactoring, good thing that linter has caught this.